documentation/src/main/asciidoc/topics/proc_creating_server_security_realms.adoc

```diff
@@ -68,6 +74,8 @@ To follow the example, specify the `*.properties` files you created with the CLI
 [discrete]
 == Multiple property realms
 
+You can define multiple security realms in an XML, a JSON, or a YAML file. The following examples demonstrate defining multiple security realms in each file type.
```

Comment:
```suggestion
The following configuration shows how you can configure multiple security realms in XML, JSON, or YAML format.
```

Reply:
Or something similar. These are not file types

Reply:
Hi @dvagnero .

Thank you for the review comment. I updated the paragraph. WDYT, keep or remove the second sentence?

Reply:
Hi @dvagnero . Just a reminder on the above ^

Reply:
I think you can shorten it and be more specific. Instead of different configuration formats you can say "in XML, JSON, and YAML file or format. One sentence is enough

Reply:
Thanks, @dvagnero . Updated.